core/templates/domain/story/story-update.service.ts

```diff
@@ -424,9 +428,11 @@ export class StoryUpdateService {
       }, (changeDict, story) => {
         // ---- Apply ----
         story.getStoryContents().markNodeOutlineAsNotFinalized(nodeId);
+        this._storyChapterUpdateEventEmitter.emit();
```

Comment:
Update the tests to test this too. Please do this for all the places.

Reply:
this is a private varaible, so i don't think this is spy able apart from this everything is tested.

Reply:
@heyimShivam Please check other spec files to  understand how to test this.

Reply:
done